Cpanel/Tests/error_log

```diff
@@ -0,0 +1,8 @@
+[15-Mar-2019 15:27:51 UTC] PHP Warning:  socket_read(): unable to read from socket [104]: Connection reset by peer in /home/phpunit/publicapi-php/Cpanel/Tests/MockSocketServer.php on line 45
```

Comment:
Did we mean include these error_Log files in the repo? I don’t see them bing used in the new tests but I suppose they might be used somewhere outside of the changes.

Reply:
Nah. They should be removed. Will action this, thank you.